Rubberduck.Core/UI/UnitTesting/TestExplorerViewModel.cs

```diff
@@ -215,6 +234,32 @@ private void HandleTestCompletion(object sender, TestCompletedEventArgs e)
             Tests.Refresh();
         }
 
+        public IRewritingManager RewritingManager { get; }
+
+        public string TestToggleLabel
+        {
+            get
+            {
+                if (IsSelectionATest)
+                {
+                    return string.Empty;
+                }
+
+                return _selectedTestMethod.IsIgnored
+                    ? RubberduckUI.TestExplorer_TestToggle_Unignore
+                    : RubberduckUI.TestExplorer_TestToggle_Ignore;
+            }
+        }
+
+        public string UnignoreTestLabel => RubberduckUI.TestExplorer_TestToggle_Unignore;
+        public string IgnoreTestLabel => RubberduckUI.TestExplorer_TestToggle_Ignore;
+
+        public bool IsSelectionATest => SelectedItem != null;
+        private TestMethod _selectedTestMethod => ((TestMethodViewModel)SelectedItem).Method;
+
+        public bool DisplayUnignoreTestLabel => IsSelectionATest && _selectedTestMethod.IsIgnored;
```

Comment:
Could be implemented as the inverse of the other property. 

`public bool DisplayUnignoreTestLabel => !DisplayIgnoreTesrLabel`

Reply:
I originally tried to set it up this way. Unfortunately it wasn't accurate and I took a while to figure out. 

This is a super fiddly bit to explain so bear with me. Below is the TestExplorer and anything within the red border will show a valid context menu. If you leave that area then [SelectedItem](https://github.com/IvenBach/Rubberduck/blob/Issue2964_TestExplorerDisableTestViaContextMenu/Rubberduck.Core/UI/UnitTesting/TestExplorerViewModel.cs#L97) will be set to null. 
![IsCursorOverTest](https://user-images.githubusercontent.com/26076874/61815006-2674d480-adfe-11e9-9b69-69d5426367e2.png)

As suggested a direct inversion of `DisplayIgnoreTestLabel` would have the `MenuItem` display when it isn't applicable, displayed below in the left-hand gutter. While it may appear as though the `MenuItem` is applicable and would visibly look like it'd apply to the last test the mouse was over `Selection` is `null` at this point and an error is thrown and caught.
![Direct inversion shows nonapplicable MenuItem](https://user-images.githubusercontent.com/26076874/61815716-bb2c0200-adff-11e9-95c2-22f13f4ee6b3.png)

For that reason the properties are written in that specific way. This does lead into an edge case I found. When you display the context menu, and while the menu is still displayed, move to another test and use `Esc` do dismiss the menu. Without moving the mouse bring up the menu again. Because the mouse wasn't moved `Selection` is still referencing the initially moused over test. Is this worth spending time fretting over?

Reply:
How about `public bool DisplayUnignoreTestLabel => Selection != null && !DisplayIgnoreTesrLabel`?